functional/MockitoTests/src/test/java/MockitoMockTest.java

```diff
@@ -0,0 +1,11 @@
+package test.java;
```

Comment:
Please add the license file. See https://github.com/adoptium/aqa-tests/blob/master/jck/jtrunner/JavatestUtil.java#L1-L13